test/models/user_test.rb

```diff
@@ -0,0 +1,29 @@
+require "test_helper"
+
+describe User do
+  describe "validation" do
```

Comment:
You should also have a test that it's valid with all the fields filled in properly.